spec/features/user_views_list_of_feelings_spec.rb

```diff
@@ -16,16 +16,15 @@
   scenario 'user sees a list' do
     user = FactoryGirl.create(:user)
 
-    visit root_path
+    visit feelings_path
 
     expect(page).to have_content('Unhappiness categories')
-    # expect(page).to have_content('Work')
-    # expect(page).to have_content('Personal life')
-    # expect(page).to have_content('Social life')
-    # expect(page).to have_content('Physical life')
-    # expect(page).to have_content('Mental life')
-    # expect(page).to have_content('Family life')
-    # expect(page).to have_content('Financial life')
-    # expect(page).to have_content('Spiritual life')
+    expect(page).to have_content('Personal life')
+    expect(page).to have_content('Social life')
+    expect(page).to have_content('Physical life')
+    expect(page).to have_content('Mental life')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.